biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/ABITrace.java

```diff
@@ -223,8 +227,12 @@ public BufferedImage getImage(int imageHeight, int widthScale) {
         Color tcolor = Color.red;
         Color ncolor = Color.pink;
         double scale = calculateScale(imageHeight);
-        int[] bc = baseCalls;
+        int[] bc = isForward ? baseCalls : getSequenceReverted(baseCalls);
         char[] seq = sequence.toCharArray();
+        int[] AS = isForward ? A : getSequenceReverted(A);
+        int[] CS = isForward ? C : getSequenceReverted(C);
+        int[] GS = isForward ? G : getSequenceReverted(G);
+        int[] TS = isForward ? T:  getSequenceReverted(T);
```

Comment:
Maybe I'm misunderstanding, but shouldn't you perform the complementation step here?

E.g. `int[] AS = isForward ? A : getSequenceReverted(T);`